packages/remix/src/generators/application/__snapshots__/application.impl.spec.ts.snap

```diff
@@ -189,7 +189,7 @@ export default defineConfig({
   },
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: 'pnpm exec nx dev test',
+    command: 'npx nx dev test',
```

Comment:
Hm... this looks different.. I think it'll fail on VSCode.. but not a blocker

Reply:
It shouldn't matter, it was picking up `pnpm-lock.yaml` from the workspace root, but since cwd changed it falls back to `npx`.